apps/mobile/src/features/feature-flags/index.ts

```diff
@@ -0,0 +1,54 @@
+import { isProduction } from '@/shared/environment';
+import { getDeviceId } from '@/utils/get-device-id';
+import { useBoolVariation } from '@launchdarkly/react-native-client-sdk';
+import { AutoEnvAttributes, ReactNativeLDClient } from '@launchdarkly/react-native-client-sdk';
+import * as Application from 'expo-application';
+
+export const featureFlagClient = new ReactNativeLDClient(
+  // TODO: do not fallback to empty string
+  process.env.EXPO_PUBLIC_LAUNCH_DARKLY ?? '',
+  AutoEnvAttributes.Enabled,
+  {
+    debug: !isProduction(),
+    applicationInfo: {
+      id: 'leather-mobile-wallet',
+      version: Application.nativeApplicationVersion ?? '0.0.1',
+    },
+  }
+);
+
+export async function setupFeatureFlags() {
+  const deviceId = await getDeviceId();
+  if (!deviceId) {
+    // TODO: handle this error properly
+    throw new Error('No device id detected');
+  }
+  featureFlagClient.identify({ kind: 'user', key: deviceId }).catch((e: any) => {
+    // TODO: analytics?
+    // eslint-disable-next-line no-console
+    console.log(e);
+  });
+}
+export function useReleaseBrowserFeatureFlag() {
+  return useBoolVariation('release_browser_feature', false);
+}
+
+export function useReleaseCollectiblesFeatureFlag() {
+  return useBoolVariation('release_collectibles_feature', false);
+}
+
+export function useReleaseLocaleFeatureFlag() {
+  return useBoolVariation('release_locale_feature', false);
+}
+
+export function useReleasePushNotificationsFlag() {
+  return useBoolVariation('release_push_notifications', false);
+}
+
+export function useReleaseNewRecipientFlowFeatureFlag() {
```

Comment:
@tigranpetrossian I renamed this hook to `useReleaseNewRecipientFlowFeatureFlag` to match the others although it's quite the mouthful. I left the key alone but maybe we should change that?